tracer/src/Datadog.Tracer.Native/iast/aspect.h

```diff
@@ -15,13 +15,23 @@ namespace iast
         Sink,
         Propagation
     };
+
+    // Keep this enum syncronized with Iast/VulnerabilityType.cs in managed code
     enum class VulnerabilityType
     {
-        None,
-        SqlInjection,
-        Xss,
-        WeakCipher,
-        WeakHash
+        None = 0,
+        WeakCipher = 1,
+        WeakHash = 2,
+        SqlInjection = 3,
+        CommandInjection = 4,
+        PathTraversal = 5,
+        LdapInjection = 6,
+        Ssrf = 7,
+        UnvalidatedRedirect = 8,
+        InsecureCookie = 9,
+        NoHttpOnlyCookie = 10,
+        NoSameSiteCookie = 11,
+        Max = 12
```

Comment:
You have all the defined enum values in `VulnerabilityType_Values` (no need to define `Max`)

Reply:
I have looked and not found an easy way to know the number of enum values in c++... Most people recommend this approach but there might be a better one with newer versions. I tried VulnerabilityType_Values but does not seem to compile.